diff '--exclude=.git' -urN a/gdk/gdkenumtypes.c b/gdk/gdkenumtypes.c
--- a/gdk/gdkenumtypes.c	2015-07-21 09:32:27.000000000 -0500
+++ b/gdk/gdkenumtypes.c	2022-12-15 14:55:57.121926392 -0600
@@ -279,6 +279,8 @@
             { GDK_TOUCH_UPDATE, "GDK_TOUCH_UPDATE", "touch-update" },
             { GDK_TOUCH_END, "GDK_TOUCH_END", "touch-end" },
             { GDK_TOUCH_CANCEL, "GDK_TOUCH_CANCEL", "touch-cancel" },
+            { GDK_TOUCHPAD_SWIPE, "GDK_TOUCHPAD_SWIPE", "touchpad-swipe" },
+            { GDK_TOUCHPAD_PINCH, "GDK_TOUCHPAD_PINCH", "touchpad-pinch" },
             { GDK_EVENT_LAST, "GDK_EVENT_LAST", "event-last" },
             { 0, NULL, NULL }
         };
@@ -304,6 +306,23 @@
 }
 
 GType
+gdk_touchpad_gesture_phase_get_type (void)
+{
+    static GType etype = 0;
+    if (G_UNLIKELY(etype == 0)) {
+        static const GEnumValue values[] = {
+            { GDK_TOUCHPAD_GESTURE_PHASE_BEGIN, "GDK_TOUCHPAD_GESTURE_PHASE_BEGIN", "begin" },
+            { GDK_TOUCHPAD_GESTURE_PHASE_UPDATE, "GDK_TOUCHPAD_GESTURE_PHASE_UPDATE", "update" },
+            { GDK_TOUCHPAD_GESTURE_PHASE_END, "GDK_TOUCHPAD_GESTURE_PHASE_END", "end" },
+            { GDK_TOUCHPAD_GESTURE_PHASE_CANCEL, "GDK_TOUCHPAD_GESTURE_PHASE_CANCEL", "cancel" },
+            { 0, NULL, NULL }
+        };
+        etype = g_enum_register_static (g_intern_static_string ("GdkTouchpadGesturePhase"), values);
+    }
+    return etype;
+}
+
+GType
 gdk_scroll_direction_get_type (void)
 {
     static GType etype = 0;
@@ -631,6 +650,7 @@
             { GDK_SCROLL_MASK, "GDK_SCROLL_MASK", "scroll-mask" },
             { GDK_TOUCH_MASK, "GDK_TOUCH_MASK", "touch-mask" },
             { GDK_SMOOTH_SCROLL_MASK, "GDK_SMOOTH_SCROLL_MASK", "smooth-scroll-mask" },
+            { GDK_TOUCHPAD_GESTURE_MASK, "GDK_TOUCHPAD_GESTURE_MASK", "touchpad-gesture-mask" },
             { GDK_ALL_EVENTS_MASK, "GDK_ALL_EVENTS_MASK", "all-events-mask" },
             { 0, NULL, NULL }
         };
